host/sgx/create.c

```diff
@@ -966,6 +1003,15 @@ oe_result_t oe_create_enclave(
     oe_enclave_t* enclave = NULL;
     oe_sgx_load_context_t context;
 
+    if (flags & OE_SGX_FLAGS_KSS && !_is_kss_supported())
```

Comment:
OE_SGX_FLAGS_KSS is not a valid flags here. See definition OE_ENCLAVE_FLAG_RESERVED.

Please move this checking to oe_sgx_build_enclave() as I commented in iteration 8.